src/GitHub/Data/Traffic.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures    #-}
+{-# LANGUAGE LambdaCase        #-}
+
+-- | Data types used in the traffic API
+module GitHub.Data.Traffic where
+
+import Data.Text   (Text)
+import Data.Time   (UTCTime)
+import Data.Vector (Vector)
+
+import GitHub.Data.Name        (Name)
+import GitHub.Internal.Prelude
+import Prelude ()
+
+data Referrer = Referrer
+    { referrer        :: !(Name Referrer)
+    , referrerCount   :: !Int
+    , referrerUniques :: !Int
+    }
+    deriving (Eq, Show, Generic)
+
+instance FromJSON Referrer where
+    parseJSON = withObject "Referrer" $ \o ->
```

Comment:
Try to replicate formatting as in https://github.com/phadej/github/blob/master/src/GitHub/Data/Issues.hs